netbench/run-netbench-request-response.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# build the scenario generation tool
+cargo build --manifest-path netbench-scenarios/Cargo.toml
+
+# generate the scenario files. This will generate .json files that can be found
+# in the netbench/target/netbench directory. Specify that there should be 1000
+# request-response occurrences
+cargo run --manifest-path netbench-scenarios/Cargo.toml -- --request_response.count 1000
```

Comment:
If we're wanting to test pure throughput it might be better to increase the request/response sizes over the count. But it just depends on what you're wanting to get out of it